Client/core/CFrustum.cpp

```diff
@@ -0,0 +1,182 @@
+#include "StdInc.h"
```

Comment:
does this really need the entire StdInc.h?

Reply:
I believe `StdInc.h` is required for precompiled headers

Reply:
I tried without but vs required stdinc.h